Tests/MapboxMapsTests/Foundation/CredentialsManagerTests.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+import MapboxMaps
+import MapboxCoreMaps
+
+class CredentialsManagerTests: XCTestCase {
+
+    private var oldDefaultAccessToken: String = ""
```

Comment:
```suggestion
    private var oldDefaultAccessToken = ""
```

can use type inference